ny-portal/src/app/(frontend)/komponenter/page.tsx

```diff
@@ -8,13 +10,25 @@ export default async function Components() {
 
     const data = await payload.find({
         collection: "component-page",
+        select: {
+            title: true,
+            slug: true,
+        },
     });
 
     return (
-        <ul>
+        <Flex as="ul" direction="column" gap={16}>
             {data.docs.map((page) => (
-                <li key={page.title}>{page.title}</li>
+                <li key={page.title}>
```

Comment:
Ikke for å flisespikke altfor mye men siden vi sørger for at `page.slug` er unik per collection er det en bedre kandidat enn `page.title`. I praksis har det neppe noe å si for jeg antar vi ikke skal ha flere kopier av samme komponent.